src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -370,6 +377,30 @@ private void validateCursors(final String topic, final List<Cursor> cursors) thr
         }
     }
 
+    public void validateCommitCursors(final String topic, final List<Cursor> cursors) throws InvalidCursorException {
```

Comment:
Why did you split this validation into two methods `validateCommitCursors` and `validateCursors`?